styles.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+* {
+  font-family: 'Crete Round', serif;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.burger {
+  display: none;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  background-color: #3e3c3c;
+  color: #fff;
+  height: 72px;
+}
+
+.brand {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  margin-left: 7%;
+}
+
+.nav-list {
+  margin-right: 17%;
+  display: flex;
+  gap: 30%;
+}
+
+.first-page {
+  background-image: url(assests/bgimg/Illustration-header\ 1.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 95vh;
+  background-color: #1c1a19;
+  background-position: right;
+  position: relative;
+}
+
+.contant {
+  color: #fff;
+  display: flex;
+  height: 95vh;
+}
+
+.social {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  gap: 25px;
+  margin-left: 2%;
+}
```

Comment:
- [ ] Your social icons width overlaps the screen width. Please fix or remove the horizontal scrolling from the body element. This will make a better user experience.

![image](https://user-images.githubusercontent.com/91486791/219038959-b1f53e06-aa4a-43b2-8f0c-b50651a2ac1f.png)